config/openshift-olm/namespace.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: netobserv
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
same thing here: it will have to be added midtream

Reply:
Actually, this `namespace` file seems to have no effect. For some reason it doesn't get added to the bundle, see the generated manifest files: there's no such namespace added to the manifests. You can remove that file and de-reference it from the `kustomize` file, and get the exact same bundle as a result.

Reply:
sure I will remove this file it seems to be used only for manual deploy